app/serializers/institution_profile_serializer.rb

```diff
@@ -92,7 +92,7 @@ class InstitutionProfileSerializer < ActiveModel::Serializer
   attribute :stem_indicator
   attribute :facility_map
   attribute :programs
-  attribute :school_certifying_officials
+  attribute :versioned_school_certifying_officials
```

Comment:
This will change your response json.  Is the frontend expecting `versioned_school_certifying_officials` or `school_certifying_officials`?

Reply:
Yes, I updated the front end to handle that change.